addOns/pscanrules/src/test/java/org/zaproxy/zap/extension/pscanrules/TimestampDisclosureScanRuleUnitTest.java

```diff
@@ -500,6 +500,22 @@ void shouldNotRaiseAlertOnValidTimeStampWhenInFontResponse(String type) throws E
         assertEquals(0, alertsRaised.size());
     }
 
+    @ParameterizedTest
+    @ValueSource(strings = {"javascript"})
+    void shouldNotRaiseAlertOnTimeStampInJavascriptFilesAtHighThreshold(String type)
+            throws Exception {
+        // Given
+        Instant testDate = ZonedDateTime.now().minusMonths(6).toInstant();
+        String strTestDate = String.valueOf(testDate.getEpochSecond());
+        HttpMessage msg = createMessage(strTestDate);
+        msg.getResponseHeader().setHeader(HttpHeader.CONTENT_TYPE, type);
+        rule.setAlertThreshold(AlertThreshold.HIGH);
+        // When
+        scanHttpResponseReceive(msg);
+        // Then
+        assertEquals(0, alertsRaised.size());
+    }
```

Comment:
Should also add a test to ensure it continues to be raised at LOW and MEDIUM.

Similar to: https://github.com/zaproxy/zap-extensions/blob/124016cff14eb93673c390353251103fd2bbd2c9/addOns/pscanrules/src/test/java/org/zaproxy/zap/extension/pscanrules/CsrfCountermeasuresScanRuleUnitTest.java#L436-L451